src/main/java/com/frostwire/jlibtorrent/TorrentStats.java

```diff
@@ -38,107 +48,103 @@ public TorrentStats(final SessionManager sessionManager, TorrentHandle torrentHa
      * reach max size denoted by the maxHistory time, they will poll() the head
      * element, in our case the oldest inserted.
      */
-    private void startAlertListener() {
-
+    public void startAlertListener() {
         sessionManager.addListener(new AlertListener() {
             @Override
             public int[] types() {
-                int[] a = new int[1];
-                a[0] = 30; //getting BLOCK_FINISHED
-                return a;
```

Comment:
here you gotta return a single element array that contains the type for the aler i showed you